bin/parse_build_gradle

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+dependencies_regex = /dependencies\s+{[\s\S]*}/
+
+dependency_metadata_regex =
+  /((implementation)|(classpath))\s['|"](?<classpath>.*):(?<package>.*):(?<version>.+)['|"]/
```

Comment:
This regex doesn't cover everything, here's an example
`        classpath("com.android.tools.build:gradle:4.1.0")`